src/Cart/CartCondition.php

```diff
@@ -113,13 +113,31 @@ public function isInclusive()
      *
      * @return float|string
      */
-    public function apply($subTotal)
+    public function apply($subTotal, string $type = null)
```

Comment:
Looks like we dont need to introduce a second parameter here, am i missing something?

Reply:
It is needed. If we dont add this code, the system would calculate `Sub Total` as `$25.47` and again apply 10% discount on the applicable menu item.

Adding this code would calculate sub total without discount(in this case $26.97). 

![image](https://github.com/tastyigniter/flame/assets/20902296/d8c3ed05-fed1-4daa-8966-9b10ebb6d709)

Reply:
Ah cool thanks for explaining